lib/rubocop/cop/style/empty_line_between_defs.rb

```diff
@@ -7,20 +7,21 @@ module Style
       # This cop checks whether method definitions are
       # separated by empty lines.
       class EmptyLineBetweenDefs < Cop
-        include OnMethodDef
         MSG = 'Use empty lines between method definitions.'.freeze
 
-        def on_method_def(node, _method_name, _args, _body)
-          return unless node.parent && node.parent.begin_type?
-
-          nodes = [prev_node(node), node]
+        def on_begin(node)
```

Comment:
I'd mention in the code why we've opted to do it like this, so someone wouldn't refactor it the way it was done before (which is pretty common in our codebase).